inc/classes/admin/settings/class-page.php

```diff
@@ -921,11 +923,32 @@ private function assets_section() {
 						'button_label' => __( 'Activate combine JavaScript', 'rocket' ),
 					],
 				],
+				'exclude_inline_js'  => [
+					'type'              => 'textarea',
+					'label'             => __( 'Excluded Inline JavaScript', 'rocket' ),
+					// translators: %1$s = opening <a> tag, %2$s = closing </a> tag.
+					'description'       => sprintf( __( 'Specify patterns of inline JavaScript to be excluded from concatenation (one per line). %1$sMore info%2$s', 'rocket' ), '<a href="' . esc_url( $inline_js_beacon['url'] ) . '" data-beacon-article="' . esc_attr( $inline_js_beacon['id'] ) . '" rel="noopener noreferrer" target="_blank">', '</a>' ),
```

Comment:
Maybe a final dot? `%1$sMore info%2$s.`

Reply:
We use the same notation in other places in the settings without the dot, so it will be either dot everywhere or dot nowhere

Reply:
OK, as long as it's the same everywhere, it's fine 👍